core/schema/src/main/resources/org/hawkular/schema/cassalog.groovy

```diff
@@ -20,4 +20,5 @@ include '/org/hawkular/schema/bootstrap.groovy'
 
 setKeyspace keyspace
 
-include '/org/hawkular/schema/updates/schema-0.15.0.groovy'
\ No newline at end of file
+include '/org/hawkular/schema/updates/schema-0.15.0.groovy'
+include '/org/hawkular/schema/updates/schema-0.16.0.groovy'
```

Comment:
Is cassalog able to skip versions? In this case you would have a file for 0.15.0 and the next one is 0.18.0.